astroquery/jplspec/tests/test_jplspec.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+
+import os
+
+from astropy import units as u
+from astropy.table import Table
+from ...jplspec import JPLSpec
+
+file1 = 'CO.data'
+file2 = 'CO_6.data'
+file3 = 'multi.data'
+
+
+def data_path(filename):
+
+    data_dir = os.path.join(os.path.dirname(__file__), 'data')
+    return os.path.join(data_dir, filename)
+
+
+class MockResponseSpec(object):
+
+    def __init__(self, filename):
+        self.filename = data_path(filename)
+
+    @property
+    def text(self):
+        with open(self.filename) as f:
+            return f.read()
+
+
+def test_input_async():
```

Comment:
As written, this looks like it's a remote query but is not tagged with `@remote_data`.  It should either be mocked so that the query is performed locally or marked `remote`.  A remote query that is not marked remote should cause the tests to fail.

Reply:
I think the intent here is to have these mocked, though.  You should follow examples from other modules, e.g. https://github.com/astropy/astroquery/blob/master/astroquery/vizier/tests/test_vizier.py#L29, to see the `pytest.fixture` mocking in action.

Reply:
We had not tagged it as remote since the get_query_payload=True bypasses the POST request and only returns the payload of the query which is the dictionary of values entered by the user, it doesn't actually make the query. We made these tests to test that the function was accurately passing the values to the dictionary that goes into the request. We mocked all the other tests because these actually require the query to  be made. We tested the script locally without --remote-data and they passed. Would we still have to mock this?